sqlglot/parser.py

```diff
@@ -6292,3 +6298,84 @@ def _parse_with_operator(self) -> t.Optional[exp.Expression]:
         op = self._parse_var(any_token=True)
 
         return self.expression(exp.WithOperator, this=this, op=op)
+
+    def _parse_copy_parameters(self) -> t.List[exp.CopyParameter]:
+        sep = TokenType.COMMA if self.dialect.COPY_PARAMS_ARE_CSV else None
```

Comment:
i don't think you need to specify dialect here, doesn't it get passed down to the class?

Reply:
the usual pattern is to do self.COPY_PARAMS_ARE_CSV

Reply:
IIRC the reason we put this in dialect is because it's used in both the parser and generator

Reply:
do we do that anywhere else? we share other things in both places right

Reply:
yep eg. we have `self.dialect.ALIAS_POST_TABLESAMPLE` both in the parser and the generator

Reply:
these dialect-wide constants are not propagated intentionally because we'd need to duplicate constant definitions in the generator, parser and tokenizer

Reply:
eg. if you make the above `self.ALIAS_POST_TABLESAMPLE` you'll get several failures:

```
    AttributeError: 'Parser' object has no attribute 'ALIAS_POST_TABLESAMPLE'
```